kubernetes/Makefile

```diff
@@ -67,6 +75,27 @@ build/concent-secrets/%: $(CONCENT_SECRET_DIR)/%
 build/%: %.j2 build/vars-merged-with-secrets.yml ../tools/render-jinja-template.sh
 	../tools/render-jinja-template.sh build/vars-merged-with-secrets.yml "$<" "$@"
 
+build/jobs/%-control-database.yml: jobs/%-database.yml.j2 ../tools/render-jinja-template.sh build/vars-with-control-database-configuration.yml
+	../tools/render-jinja-template.sh build/vars-with-control-database-configuration.yml "$<" "$@"
+
+build/jobs/%-storage-database.yml: jobs/%-database.yml.j2 ../tools/render-jinja-template.sh build/vars-with-storage-database-configuration.yml
+	../tools/render-jinja-template.sh build/vars-with-storage-database-configuration.yml "$<" "$@"
+
+build/secrets/%-db-secrets.yml: build/vars-with-%-database-configuration.yml secrets/db-secrets.yml.j2 ../tools/render-jinja-template.sh
+	../tools/render-jinja-template.sh "$<" "secrets/db-secrets.yml.j2" "$@"
+
+
+
+build/vars-with-%-database-configuration.yml: build/%-database-configuration.yml var.yml var-$(CLUSTER).yml $(CONCENT_SECRET_DIR)/var-secret.yml build/image-version.yml
+	../tools/merge-yaml.py                         \
```

Comment:
You could merge `build/%-database-configuration.yml` with `build/vars-merged-with-secrets.yml` to avoid repetition instead of listing all the yaml files here.

Also, for consistency, please add `-with-secrets` to the name of this config too. It makes the name long but without it it might not be clear that it contains secrets. Maybe use `config` instead of `configuration` to make it a bit shorter.

Reply:
Done.